apps/backend/src/modules/auth/jwt-auth/auth.jwt.ts

```diff
@@ -0,0 +1,22 @@
+import { SignJWT } from "jose";
+
+const rawSecret = process.env["JWT_SECRET"];
+
+if (!rawSecret) {
+	throw new Error("Missing JWT_SECRET in .env file");
+}
+
+const secret = new TextEncoder().encode(rawSecret);
+
+async function generateJWT(userId: string): Promise<string> {
+	const token = await new SignJWT({})
+		.setProtectedHeader({ alg: "HS256" })
+		.setSubject(userId)
+		.setIssuedAt()
+		.setExpirationTime("1d")
+		.sign(secret);
+
+	return token;
+}
+
+export { generateJWT };
```

Comment:
Also your service must include a method to decod JWT token to get the user data out of there